core/src/test/scala/scalacss/RandomData.scala

```diff
@@ -34,16 +34,12 @@ object RandomData {
       LastOfType, Link, OnlyOfType, OnlyChild, Optional, OutOfRange, ReadOnly, ReadWrite, Required, Target, Valid,
       Visited, After, Before, FirstLetter, FirstLine, Selection)
 
-    val needInt = NonEmptyList[Int => Pseudo](
-      NthChild, NthLastChild, NthLastOfType, NthOfType)
-
     val needStr = NonEmptyList[String => Pseudo](
       Custom(_, PseudoClass), Custom(_, PseudoElement), Lang, Not(_))
 
     lazy val self: Gen[Pseudo] =
       Gen.frequency[Pseudo](
         objects.size -> Gen.oneofL(objects),
-        needInt.size -> Gen.oneofL(needInt).flatMap(Gen.positiveint.map),
```

Comment:
Btw, I've just updated `RandomData.scala` so make sure you do a merge or rebase before commencing work. Cheers.